osu.Game/Overlays/Music/PlaylistList.cs

```diff
@@ -124,6 +127,31 @@ public BeatmapSetInfo SelectedSet
             public BeatmapSetInfo FirstVisibleSet => items.FirstOrDefault(i => i.MatchingFilter)?.BeatmapSetInfo;
             public BeatmapSetInfo NextSet => (items.SkipWhile(i => !i.Selected).Skip(1).FirstOrDefault() ?? items.FirstOrDefault())?.BeatmapSetInfo;
             public BeatmapSetInfo PreviousSet => (items.TakeWhile(i => !i.Selected).LastOrDefault() ?? items.LastOrDefault())?.BeatmapSetInfo;
+            public BeatmapSetInfo RandomSet()
+            {
+                randomSelectedSets.Push(SelectedSet);
+                return items[RNG.Next(items.Count - 1)]?.BeatmapSetInfo;
+            }
+            public BeatmapSetInfo LastPlayedRandomSet()

```

Comment:
what is this doing?

Reply:
For example, if we choose a random song, then choose another random song and we want to go back to the first one